test/expected/agg_bookends.out

```diff
@@ -0,0 +1,110 @@
+\ir include/create_single_db.sql
+SET client_min_messages = WARNING;
+DROP DATABASE IF EXISTS single;
+SET client_min_messages = NOTICE;
+CREATE DATABASE single;
+\c single
+CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
+psql:include/create_single_db.sql:7: NOTICE:  installing required extension "postgres_fdw"
+SELECT setup_timescaledb(hostname => 'fakehost'); -- fakehost makes sure there is no network connection
+ setup_timescaledb 
+-------------------
+ 
+(1 row)
+
+CREATE TABLE "btest"(time timestamp, gp INTEGER, temp float, strid TEXT DEFAULT 'testing');
+SELECT create_hypertable('"btest"', 'time');
+ create_hypertable 
+-------------------
+ 
+(1 row)
+
+INSERT INTO "btest" VALUES('2017-01-20T09:00:01',1, 22.5);
+INSERT INTO "btest" VALUES('2017-01-20T09:00:21',1, 21.2);
+INSERT INTO "btest" VALUES('2017-01-20T09:00:47',1, 25.1);
+INSERT INTO "btest" VALUES('2017-01-20T09:00:02',2, 35.5);
+INSERT INTO "btest" VALUES('2017-01-20T09:00:21',2, 30.2);
+--TOASTED;
+INSERT INTO "btest" VALUES('2017-01-20T09:00:43',2, 20.1, repeat('xyz', 1000000) );
```

Comment:
Anyway to verify that this column will be toasted, apart from overrunning the length limit when inserting?